include/boost/beast/http/impl/file_body_win32.hpp

```diff
@@ -337,6 +337,46 @@ class null_lambda
     }
 };
 
+// https://github.com/boostorg/beast/issues/1815
+// This function mimics the behaviour of ASIO.
```

Comment:
I would write
```
// RHODGES This function mimics the behaviour of ASIO....
```
To make it clear that this is developer commentary and not documentation about the code

Reply:
This is how I get blamed for things I didn't say :-)

Reply:
Alternatively you could use VFALCO